server/src/instant/flags.clj

```diff
@@ -121,6 +122,24 @@
              :default-value default-value
              :disabled? disabled?}))
 
+        store-fair-lock
```

Comment:
Thought: would the nifty `flags` namespace that @dwwoelfel popped up recently be useful for this? We could keep all of this in a json field there, so we don't have to create a new namespace

Reply:
I like the flexibility of disabled/disabled-apps/enabled-apps/enabled. Maybe we can create a universal namespace for this. I’ll do it next time